docs/install.rst

```diff
@@ -32,5 +32,5 @@ Installing from GitHub
     * pip install .
 
     * verify installation is complete via:  ``pip list | grep pyprof``. 
-        # it should display ``pyprof 0.1``
```

Comment:
This is just as bad, but with a redirection. Note each release branch will need to change this link to point to the correct release. At that point, just leave it as is and update when we update versions as part of the release. Or add something more generic like

```
   # Should display ``pyprof x.y`` where ``x.y`` is the PyProf version installed.
```

Reply:
The intention was to point the version to the file we are regularly updating. I do prefer `the x.y` wording instead since the version number itself is not important, just the grep returns a value.